update Baking with the possibility of staking (#463)
trilitech/tezos-developer-docs: docs/overview/tezos-different.md

## Tezos uses proof of stake

> The proof-of-stake consensus model eliminates the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work models, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate and stake tokens to a baker for a share of the rewards. The bakers keep control of their tokens in that they can unstake them later; and delegators keep complete control of their tokens. Tezos's approach to consensus has been described as [Liquid Proof of Stake](https://medium.com/tezos/liquid-proof-of-stake-aec2f7ef1da7).

Reviewer: ```suggestion
Proof-of-stake consensus mechanism eliminate the need for high energy use, making it the "green" choice for blockchains. Instead of competing to achieve consensus as in proof-of-work mechanisms, Tezos nodes (called *bakers*) stake Tezos tokens to earn the right to create blocks and receive rewards. Users who want to participate without running a node themselves can delegate and stake tokens to a baker for a share of the rewards. Bakers and stakers keep control of their tokens in that they can unstake them later; and delegators keep complete control of their tokens. ```

> The proof-of-stake model improves scalability and encourages cooperation via incentives. It also increases the cost of 51% attacks and avoids environmentally wasteful proof-of-work. Tezos launched in June 2018 as one of the first major proof-of-stake networks.

Reviewer: ```suggestion
Tezos' proof-of-stake mechanism improves scalability and encourages cooperation via incentives. It also increases the cost of consensus attacks and avoids environmentally wasteful proof-of-work. Tezos launched in June 2018 as one of the first major proof-of-stake networks.
```

I'm not sure I buy the scalability argument. we wouldn't need L2s for PoS blockchains if that were the case. Also, we need to be careful of 51% attacks, when we have a BFT consensus algorithm. Some attacks need 67% of the stake to take control, but it is cheaper to block consensus (by shutting down 34% of the stake)
